packages/webapp/src/components/Map/PreviewPopup/utils.js

```diff
@@ -1,5 +1,5 @@
-const metric = ['metric', 'celsius'];
-const imperial = ['imperial', 'fahrenheit'];
+const metric = ['metric', 'celsius', 'kpa'];
+const imperial = ['imperial', 'fahrenheit', 'psi'];
 
```

Comment:
temperature is already in convert-unit library
https://www.npmjs.com/package/convert-units#temperature
Also, the library support custom units
https://github.com/convert-units/convert-units

Reply:
The implementation of temperature conversion was added by sein. I have just worked on the soil water potential